modules/oxide/src/less/theme/components/menubar/promotion.less

```diff
@@ -0,0 +1,46 @@
+//
+// Promotion
+//
+
+@promotion-background-color: #E8F1F8;
+@promotion-background-hover-color: #B4D7FF;
+@promotion-background-active-color: #D9EDF7;
+@promotion-color: #086BE6;
```

Comment:
This should probably be called `promotion-text-color` for consistency with other variables and it also makes it clearer what it applies to.